.github/release-template.md

```diff
@@ -38,5 +38,5 @@ docker pull ghcr.io/darwinia-network/darwinia-2.0:sha-{{ sha }}
 | Pangoro  | `{{ pangoro_proposal_compressed }}`  | `{{ pangoro_runtime_version }}`  |
 
 ## Changelog
-[Darwinia 2.0](https://github.com/darwinia-network/darwinia-2.0/issues/9)
+[Darwinia 2.0](https://github.com/darwinia-network/darwinia/issues/9)

```

Comment:
Please changes this link to https://github.com/darwinia-network/darwinia/pull/969

Reply:
Anyway, this will be updated in the final release PR.